base/loading.jl

```diff
@@ -638,15 +638,17 @@ function find_source_file(path::AbstractString)
     return isfile(base_path) ? base_path : nothing
 end
 
-cache_file_entry(pkg::PkgId) =
-    pkg.uuid === nothing ? "$(pkg.name).ji" :
-        joinpath(pkg.name, "$(package_slug(pkg.uuid)).ji")
+cache_file_entry(pkg::PkgId) = joinpath(
+    "compiled",
```

Comment:
Why “compiled”?

Reply:
Because this is where things that are compiled live. The option that controls whether julia creates or uses these files is called `--compiled-modules`. (There was also a [bikeshed issue](https://github.com/JuliaLang/Pkg3.jl/issues/142) where I asked for comment on this, which was posted in multiple forums.)